### ISSUE TITLE
youtube integration

### ISSUE THREAD

**Issue by waterskier2007** [StreamCenter/StreamCenter#39]

**Comment by waterskier2007** [comment]
Where do we stand on this?

**Comment by olivierboucher** [comment]
Next thing I do is hitbox chat ! This integration is tricky because you have to deal with youtube api and there's nothing regarding gaming in general.

**Comment by waterskier2007** [comment]
Yep, and they don't expose any links other than their direct embed links, which we can't do because there is no web view on apple tv. still poking around it a bit
